src/components/keypad/button.js

```diff
@@ -0,0 +1,87 @@
+import React from "react";
+
+import {StyleSheet} from "aphrodite";
+import {View} from "@khanacademy/wonder-blocks-core";
+import Color from "@khanacademy/wonder-blocks-color";
+
+import Clickable from "@khanacademy/wonder-blocks-clickable";
+
+type Props = {
```

Comment:
Should the type be exact? e.g. `type Props = {| ... }}`?

Reply:
Yep! I'll address that in PR #64